.github/workflows/go.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         echo "=========START LINT============"
         golint -set_exit_status ./cmd/... ./internal/...
         echo "=========START TESTS IN OVS============"
-        go test -v -race ./ovs/
+        go test -v -race -short ./ovs/
```

Comment:
what does that change ?

Reply:
It is to not run my tests with binary but only my mocked tests